kano_updater/scenarios.py

```diff
@@ -359,4 +359,22 @@ def beta_210_to_beta_220(self):
             )
 
     def beta_220_to_beta_230(self):
-        pass
+        os.system('sudo adduser {} i2c'.format(get_user_unsudoed()))
```

Comment:
Would it be possible to apply this over all normal users currently on the system? We want all of our previously created users to be able to use the speaker led deamon without sudo.